src/modules/assessments/components/AssessmentFormSideBar.tsx

```diff
@@ -79,35 +105,45 @@ const AssessmentFormSideBar: React.FC<Props> = ({
         scrollOffset={top}
         useUrlHash={!embeddedInWorkflow}
       />
-      <Divider sx={{ my: 2, mx: -2 }} />
-      <Stack gap={2} sx={{ mt: 2 }}>
-        {showAutofill && <AssessmentAutofillButton onClick={onAutofill} />}
-        {!isPrintView && locked && assessment && (
-          <PrintViewButton
-            openInNew
-            to={generateSafePath(EnrollmentDashboardRoutes.VIEW_ASSESSMENT, {
-              clientId: assessment.enrollment.client.id,
-              enrollmentId: assessment.enrollment.id,
-              assessmentId: assessment.id,
-            })}
-          >
-            Print
-          </PrintViewButton>
-        )}
-        {assessment && (
-          <DeleteAssessmentButton
-            assessment={assessment}
-            clientId={enrollment.client.id}
-            enrollmentId={enrollment.id}
-            onSuccess={navigateToEnrollment}
-          />
-        )}
-        {assessment && import.meta.env.MODE === 'development' && (
-          <Typography variant='body2'>
-            <b>Assessment ID:</b> {assessment.id}
-          </Typography>
-        )}
-      </Stack>
+      {(showAutofill ||
+        showPrintViewButton ||
+        showDeleteAssessmentButton ||
+        showAssessmentId) && (
+        <>
+          <Divider sx={{ my: 2, mx: -2 }} />
+          <Stack gap={2} sx={{ mt: 2 }}>
+            {showAutofill && <AssessmentAutofillButton onClick={onAutofill} />}
+            {showPrintViewButton && (
+              <PrintViewButton
+                openInNew
+                to={generateSafePath(
+                  EnrollmentDashboardRoutes.VIEW_ASSESSMENT,
+                  {
+                    clientId: assessment.enrollment.client.id,
+                    enrollmentId: assessment.enrollment.id,
+                    assessmentId: assessment.id,
+                  }
+                )}
+              >
+                Print
+              </PrintViewButton>
+            )}
+            {showDeleteAssessmentButton && assessment && (
```

Comment:
I added `&& assessment` here to keep typescript happy, even though it's always true from `showDeleteAssessmentButton`. The other thing we could do is pass `assessment as FullAssessmentFragment`. Let me know if you prefer one over the other!

Reply:
I usually take the approach you have here to keep typescript happy when necessary